finite_state_sdk/queries.py

```diff
@@ -627,6 +724,81 @@ def _create_LAUNCH_CYCLONEDX_EXPORT_VARIABLES(cdx_subtype, asset_version_id):
 }
 
 
+def _create_LAUNCH_REPORT_EXPORT_MUTATION(asset_version_id=None, product_id=None, report_type=None, report_subtype=None):
+    if not asset_version_id and not product_id:
+        raise Exception("Must specify either asset_version_id or product_id")
+
+    if asset_version_id and product_id:
+        raise Exception("Cannot specify both asset_version_id and product_id")
+
+    if asset_version_id:
+        if report_type == "CSV":
```

Comment:
Please consider replacing strings with Enum values for improved code clarity and maintainability.